controllers/patchArticles.controller.js

```diff
@@ -0,0 +1,32 @@
+const updateArticles = require('../models/patchArticles.model')
+const getArticleByID = require('../models/getArticleByID.model')
+
+function patchArticles (req, res, next) {
+    const article_id = req.params.article_id
+    const {inc_votes} = req.body
+    if (inc_votes === undefined || typeof inc_votes !== 'number'){
```

Comment:
Same here please move any logic to the model. There is more similar logic later in this controller and others. Please move all logic to the model.